src/Domain/CoordinateMaps/CubicScale.hpp

```diff
@@ -0,0 +1,93 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <limits>
+#include <string>
+#include <unordered_map>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/TypeTraits.hpp"
+/// \cond
+class FunctionOfTime;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace CoordMapsTimeDependent {
+/*!
+ * \ingroup CoordMapsTimeDependentGroup
+ * \brief CubicScale map defined by \f$x = a(t)*\xi+(b(t)-a(t))\xi^3/X^2\f$,
+ *  where \f$X\f$ is the outer boundary.
+ *
+ * The map scales the coordinates \f$\xi\f$ near the center by a factor
+ * \f$a(t)\f$, while the outer the boundary \f$X\f$, is scaled by a
```

Comment:
It sounds like there are two kinds of scaling being used here - the Jacobian scales the coordinates by a(t) near the origin of the source frame, but not by b(t) near X. The origin is not mapped to a(t), but x(X) is mapped to b(t)*X.